.kitchen.yml

```diff
@@ -53,3 +53,16 @@ suites:
       consul:
         service_mode: cluster
         bootstrap_expect: 1
+  - name: atlas
+    run_list:
+      - recipe[consul::default]
+    attributes:
+      consul:
+        atlas_autojoin: true
+        atlas_cluster: <%= ENV.fetch('ATLAS_CLUSTER', 'example/cluster') %>
+        atlas_token: <%= ENV.fetch('ATLAS_TOKEN', 'NOT_REAL') %>
+    excludes:
+      - centos-7.0
```

Comment:
If I recall, I believe CentOS 5.x and 7.x were stumbling on the libarchive dependency when I tried, perhaps due to package naming. They seemed to fail similarly on master without my changes so I haven't started digging yet.